src/components/SystemIntelligences.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <section
+    v-show="systems.includes($route.name)"
+    class="container"
+  >
+    <img
+      v-if="loading"
+      class="logo"
+      src="../assets/LogoWeniAnimada4.svg"
+    />
+
+    <iframe
+      ref="iframe"
+      @load="load"
+      v-show="!loading"
+      class="container"
+      allow="clipboard-read; clipboard-write;"
+      frameborder="0"
+      :style="{ height: '100%' }"
```

Comment:
What is a need for this style in line? Why aren't you with the 'container' stylization?